Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -229,34 +236,194 @@ extension NavigationView {
         navigationMapView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
         navigationMapView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
   
-        topBannerContainerView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
-        topBannerContainerView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
-        topBannerContainerView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        let regularTopBannerContainerViewLeadingConstraint = topBannerContainerView.leadingAnchor.constraint(equalTo: leadingAnchor)
+        let regularTopBannerContainerViewTrailingConstraint = topBannerContainerView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        let regularTopBannerContainerViewTopConstraint = topBannerContainerView.topAnchor.constraint(equalTo: topAnchor)
+        
+        let regularTopBannerConstraints = [
+            regularTopBannerContainerViewLeadingConstraint,
+            regularTopBannerContainerViewTrailingConstraint,
+            regularTopBannerContainerViewTopConstraint
+        ]
+        
+        let screenWidth = UIScreen.main.bounds.width
+        let compactTopBannerContainerViewLeadingConstraint = topBannerContainerView.leadingAnchor.constraint(equalTo: leadingAnchor)
+        let compactTopBannerContainerViewTrailingConstraint = topBannerContainerView.trailingAnchor.constraint(equalTo: trailingAnchor,
+                                                                                                               constant: -(screenWidth * 0.6))
```

Comment:
The top banner and bottom banner each have a `SeparatorView` to better delineate the top and bottom banners from the map when the banner background color has low contrast with the map style’s background color. Can we add a similar divider to the trailing side of the banner views?

Reply:
Yep, will add.